ledger/ledgertestutils/ledgertestutils.go

```diff
@@ -22,36 +22,30 @@ import (
 	"github.com/insolar/insolar/configuration"
 	"github.com/insolar/insolar/core"
 	"github.com/insolar/insolar/instrumentation/inslogger"
+	"github.com/insolar/insolar/ledger"
 	"github.com/insolar/insolar/ledger/artifactmanager"
 	"github.com/insolar/insolar/ledger/jetcoordinator"
 	"github.com/insolar/insolar/ledger/localstorage"
 	"github.com/insolar/insolar/ledger/pulsemanager"
-	"github.com/insolar/insolar/network/nodenetwork"
-	"github.com/stretchr/testify/assert"
-
-	"github.com/insolar/insolar/ledger"
 	"github.com/insolar/insolar/ledger/storage/storagetest"
+	"github.com/insolar/insolar/log"
+	"github.com/insolar/insolar/network/nodenetwork"
 	"github.com/insolar/insolar/testutils/testmessagebus"
 )
 
-// TmpLedger crteates ledger on top of temporary database.
-// Returns *ledger.Ledger andh cleanup function.
+// FIXME: THIS METHOD IS DEPRECATED. USE MOCKS.
```

Comment:
wrong godoc (you ci=uld add `fixme` part inside it, but don't remove actual doc)